packages/rust-bindings/CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # Changelog
 
+## Unreleased
+
+### Breaking changes
+
+- Errors returned from entrypoints in `external_functions.rs` are now proper JS errors.
```

Comment:
I would say that this is not a breaking change :upside_down_face:

Reply:
Well it is though as the type changes from `string` to `Error`? If all you do is print the error, all that will happen is that the message has "Error: " prepended, however we cannot really be sure. Isn't it best to just be honest about this?